common/models/workspace.js

```diff
@@ -215,6 +215,14 @@ module.exports = function(Workspace) {
             .filter(function(cc) { return cc.name != explorer; });
       }
 
+      // Remove legacyExplorer from LoopBack 3.x app template
```

Comment:
We should remove the option for versions beyond 3.x, e.g. 4.x too.

I think a better solution is to remove `legacyExplorer` from the template, and then add it here if `loopbackVersion === '2.x'`.

Thoughts?